src/Microsoft.Cci.Extensions/Writers/CSharp/CSDeclarationWriter.Fields.cs

```diff
@@ -69,5 +73,28 @@ private void WriteFieldDefinition(IFieldDefinition field)
                 WriteSymbol(",");
             }
         }
+
+        private void WriteFieldDefinitionValue(IFieldDefinition field) {
+            var resolvedType = field.Type.ResolvedType;
+
+            if (resolvedType != null) {
+                foreach (var enumField in resolvedType.Fields) {
+                    var enumFieldValue = enumField?.Constant?.Value;
+                    if (enumFieldValue != null && enumFieldValue.Equals(field.Constant.Value))
+                    {
+                        WriteTypeName(field.Type, true);
```

Comment:
Can you specify which "true" is as the second parameter? based on the code it looks like it would be for nospace.